Source/JavaScriptCore/wasm/WasmAirIRGenerator.cpp

```diff
@@ -594,7 +594,48 @@ class AirIRGenerator {
             return { };
         }
         if (isValidForm(airOp, Arg::RelCond, Arg::SIMDInfo, Arg::Tmp, Arg::Tmp, Arg::Tmp)) {
+#if CPU(X86_64)
```

Comment:
We usually use `if (isX86())` in WasmAirIRGenerator to keep both branches compiled / syntax-checked. (And it is constexpr).

Reply:
I'll switch it over, thanks for the review!